### ISSUE TITLE
Komodo bleeding fails with import error

### ISSUE THREAD

**Issue by equinor-ruaj** [equinor/fmu-sumo-sim2sumo#8]
```
 ImportError: cannot import name 'main' from 'fmu.sumo.sim2sumo'
```

Related [slack thread](https://equinor.slack.com/archives/C01V40WH48J/p1698403157663579)

**Comment by equinor-ruaj** [comment]
Should have been fixed by #5.

Looking into it we found that the last successful Komodo bleeding build was 25.10.23, which was before #5 was merged.
Most likely the issue will be resolved when Komodo bleeding builds again. (As far as we can tell, the Komodo bleeding build is not failing due to any of our repos)

**Comment by daniel-sol** [comment]
fixed with pull #5
